### ISSUE TITLE
Does not compile with DMD 2.068

### ISSUE THREAD

**Issue by maaaks** [dymk/temple#29]
Hi, after upgrading compiler to 2.068 I can't use Temple. Because of error in this line:
https://github.com/dymk/temple/blob/master/src/temple/package.d#L215
```d
static assert(isOutputRange!(vibe.core.stream.OutputStream, string));
```
> Error: static assert  (isOutputRange!(OutputStream, string)) is false

BTW, please fix line number and filename in this line:
```d
#line 75 "src/temple/temple.d"
```

**Comment by DarkRiDDeR** [comment]
line 215 "src/temple/temple.d"
```
static assert(isOutputRange!(vibe.core.stream.OutputStream, string));
```
replace to:
```
static assert(isOutputRange!(vibe.stream.wrapper.StreamOutputRange, string));
```

**Comment by rnine** [comment]
@DarkRiDDeR :+1:

**Comment by denizzzka** [comment]
Same bug or not?

https://github.com/dymk/temple/issues/28

**Comment by dymk** [comment]
Fixed with #30

**Comment by dymk** [comment]
Ignore the "Build Failing", that's just Travis running out of memory when compiling temple+vibe. I'm working on moving to their new infrastructure which has more memory.
